nearai/aws_runner/Dockerfile

```diff
@@ -1,14 +1,36 @@
 FROM public.ecr.aws/lambda/python:3.11
 
-RUN yum install -y gcc libffi-devel openssl-devel
+RUN yum install -y gcc libffi-devel openssl-devel shadow-utils acl
 
 # pass with --build-arg FRAMEWORK=-langgraph
 ARG FRAMEWORK=""
+ARG AGNER_RUNNER_USER="agent_runner"
+ARG HUB_CONFIG_PATH="/opt/hub_config.env"
 
 COPY nearai/aws_runner/frameworks/requirements${FRAMEWORK}.txt ${LAMBDA_TASK_ROOT}
 
 RUN pip install -r requirements${FRAMEWORK}.txt
 
+CMD /bin/sh -c "echo Setting agent runner user as $AGNER_RUNNER_USER"
+
+RUN /usr/sbin/groupadd -r $AGNER_RUNNER_USER && \
+    /usr/sbin/useradd -r -g $AGNER_RUNNER_USER -m $AGNER_RUNNER_USER
+
+RUN mkdir -p /$AGNER_RUNNER_USER/.nearai/registry && \
+    chown $AGNER_RUNNER_USER:$AGNER_RUNNER_USER /$AGNER_RUNNER_USER/.nearai/registry
+
+COPY nearai/aws_runner/.env $HUB_CONFIG_PATH
```

Comment:
Since the `nearai` folder gets copied in on line 33 we'll need to move aws_runner back out of nearai (which we should do anyway).